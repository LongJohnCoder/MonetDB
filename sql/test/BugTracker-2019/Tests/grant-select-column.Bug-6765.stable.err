stderr of test 'grant-select-column.Bug-6765` in directory 'sql/test/BugTracker-2019` itself:


# 22:34:40 >  
# 22:34:40 >  "/usr/bin/python3" "grant-select-column.Bug-6765.py" "grant-select-column.Bug-6765"
# 22:34:40 >  

MAPI  = (myuser) /var/tmp/mtest-26496/.s.monetdb.31053
QUERY = SELECT "id", "name" FROM "myschema"."test"; --error, no permission on column "name"
ERROR = !SELECT: identifier 'id' unknown
CODE  = 42000
# builtin opt 	gdk_dbpath = /home/niels/scratch/rc-nov2019/Linux-x86_64/var/monetdb5/dbfarm/demo
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_ipv6 = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 31053
# cmdline opt 	mapi_usock = /var/tmp/mtest-26496/.s.monetdb.31053
# cmdline opt 	gdk_dbpath = /home/niels/scratch/rc-nov2019/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2019
#client4:!ERROR:ParseException:SQLparser:42000!SELECT: identifier 'id' unknown

# 22:34:40 >  
# 22:34:40 >  "Done."
# 22:34:40 >  

