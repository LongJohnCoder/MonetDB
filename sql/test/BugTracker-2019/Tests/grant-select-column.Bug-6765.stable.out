stdout of test 'grant-select-column.Bug-6765` in directory 'sql/test/BugTracker-2019` itself:


# 19:45:13 >  
# 19:45:13 >  "/usr/bin/python3" "grant-select-column.Bug-6765.py" "grant-select-column.Bug-6765"
# 19:45:13 >  

#CREATE schema  "myschema";CREATE TABLE "myschema"."test" ("id" integer, "name" varchar(20));INSERT INTO "myschema"."test" ("id", "name") VALUES (1,'Tom'),(2,'Karen');CREATE USER myuser WITH UNENCRYPTED PASSWORD 'Test123' NAME 'Hulk' SCHEMA "myschema";GRANT SELECT ON "myschema"."test" TO myuser;
#CREATE schema  "myschema";CREATE TABLE "myschema"."test" ("id" integer, "name" varchar(20));INSERT INTO "myschema"."test" ("id", "name") VALUES (1,'Tom'),(2,'Karen');CREATE USER myuser WITH UNENCRYPTED PASSWORD 'Test123' NAME 'Hulk' SCHEMA "myschema";GRANT SELECT ON "myschema"."test" TO myuser;
#CREATE schema  "myschema";CREATE TABLE "myschema"."test" ("id" integer, "name" varchar(20));INSERT INTO "myschema"."test" ("id", "name") VALUES (1,'Tom'),(2,'Karen');CREATE USER myuser WITH UNENCRYPTED PASSWORD 'Test123' NAME 'Hulk' SCHEMA "myschema";GRANT SELECT ON "myschema"."test" TO myuser;
[ 2	]
#CREATE schema  "myschema";CREATE TABLE "myschema"."test" ("id" integer, "name" varchar(20));INSERT INTO "myschema"."test" ("id", "name") VALUES (1,'Tom'),(2,'Karen');CREATE USER myuser WITH UNENCRYPTED PASSWORD 'Test123' NAME 'Hulk' SCHEMA "myschema";GRANT SELECT ON "myschema"."test" TO myuser;
#CREATE schema  "myschema";CREATE TABLE "myschema"."test" ("id" integer, "name" varchar(20));INSERT INTO "myschema"."test" ("id", "name") VALUES (1,'Tom'),(2,'Karen');CREATE USER myuser WITH UNENCRYPTED PASSWORD 'Test123' NAME 'Hulk' SCHEMA "myschema";GRANT SELECT ON "myschema"."test" TO myuser;
#SELECT "id", "name" FROM "myschema"."test";
% myschema.test,	myschema.test # table_name
% id,	name # name
% int,	varchar # type
% 1,	5 # length
[ 1,	"Tom"	]
[ 2,	"Karen"	]
#REVOKE SELECT ON "myschema"."test" FROM myuser;GRANT SELECT ("name") ON "myschema"."test" TO myuser;
#REVOKE SELECT ON "myschema"."test" FROM myuser;GRANT SELECT ("name") ON "myschema"."test" TO myuser;
#DROP USER myuser;DROP SCHEMA "myschema" CASCADE;
#DROP USER myuser;DROP SCHEMA "myschema" CASCADE;
# MonetDB 5 server v11.36.0 (hg id: 1d18bd8860b4+)
# This is an unreleased version
# Serving database 'mTests_sql_test_BugTracker-2019', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 15.384 GiB available main-memory.
# Copyright (c) 1993 - July 2008 CWI.
# Copyright (c) August 2008 - 2019 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://xps13:33829/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-3703/.s.monetdb.33829
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

# 19:45:13 >  
# 19:45:13 >  "Done."
# 19:45:13 >  

